server/storage/players.lua

```diff
@@ -198,6 +199,7 @@ local function fetchAllPlayerEntities(license2, license)
         chars[i].gang = result[i].gang and json.decode(result[i].gang)
         chars[i].position = convertPosition(result[i].position)
         chars[i].metadata = json.decode(result[i].metadata)
+        chars[i].lastLoggedOut = result[i].lastLoggedOut and result[i].lastLoggedOut/1000
```

Comment:
math.floor the division?

Reply:
no need to floor a whole number

Reply:
It isn't a whole number always when you divide

Reply:
Ok nvm in this case we shouldn't even divide since MYSQL.query returns Milliseconds and not microseconds for some reason